app/routes/endemics/you-can-claim-multiple.js

```diff
@@ -0,0 +1,71 @@
+const session = require('../../session')
+const { agreeSameSpecies, organisation: organisationKey } = require('../../session/keys').farmerApplyData
+const config = require('../../config/index')
+const urlPrefix = require('../../config/index').urlPrefix
+const {
+  endemicsNumbers,
+  endemicsYouCanClaimMultiple,
+  endemicsCheckDetails,
+  endemicsOfferRejected
+} = require('../../config/routes')
+
+const pageUrl = `${urlPrefix}/${endemicsYouCanClaimMultiple}`
+const backLink = `${urlPrefix}/${endemicsCheckDetails}`
+const nextPage = `${urlPrefix}/${endemicsNumbers}`
+
+const agreeStatusValue = 'agree'
+const agreementStatuses = [
+  {
+    value: agreeStatusValue,
+    text: 'I agree',
+    storedAnswer: 'yes'
```

Comment:
Not sure I understand why we use a different value for the form components from what we store. Would it work if we just set the values for the two buttons to be 'yes' and 'no'?

Reply:
Agreed, was on my list but forgot to action, will get back to you but can't see why they would need to be different

Reply:
Updated code to use button value instead of storedAnswer